<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inserisci_prenotazione</class>
 <widget class="QWidget" name="inserisci_prenotazione">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuova Prenotazione</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(69, 162, 0);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayoutCliente">
        <item>
         <widget class="QListView" name="listViewClienti">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="labelCliente">
            <property name="text">
             <string>Inserisci il cliente:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textCliente">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="RicercaButton_Cliente">
            <property name="styleSheet">
             <string notr="true">color: rgb(69, 162, 0);
background-color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Ricerca cliente</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayoutCampo">
           <item>
            <widget class="QListView" name="listViewCampi">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="labelCampo">
               <property name="text">
                <string>Inserisci il campo:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="textCampo">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="RicercaButton_Campo">
               <property name="styleSheet">
                <string notr="true">color: rgb(69, 162, 0);
background-color: rgb(0, 0, 0);</string>
               </property>
               <property name="text">
                <string>Ricerca campo</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(69, 162, 0);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="labelCalendario">
            <property name="text">
             <string>Scegli giorno e ora:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCalendarWidget" name="calendarWidget">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);
color: rgb(69, 162, 0);
selection-background-color: rgb(0, 76, 255);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_ok">
            <property name="styleSheet">
             <string notr="true">color: rgb(69, 162, 0);
background-color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>OK</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QListView" name="listViewOrari">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QWidget" name="widget_4" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(69, 162, 0);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QPushButton" name="Conferma_button">
        <property name="styleSheet">
         <string notr="true">color: rgb(69, 162, 0);
background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Prenota senza servizio</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="Conferma_assegnamento">
        <property name="styleSheet">
         <string notr="true">color: rgb(69, 162, 0);
background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Aggiungi servizio</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
